src/strategies/eoa-balance-and-staking-pools/index.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable prettier/prettier */
+import { Contract } from '@ethersproject/contracts';
+import { JsonRpcProvider } from '@ethersproject/providers';
+import { formatEther } from '@ethersproject/units';
+
+export const author = 'bonustrack';
+export const version = '0.1.1'; 
+
+
+const tokenAbi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+const abiStakingPool = [
+  'function getSmartPoolVotes (address account, uint256 blockNumber) public view returns (uint256 votes)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
```

Comment:
you can use this provider instead of creating a new instance of `JsonRpcProvider`